### PR TITLE
PR changing config.js to enable "previous rec" and  the link to the errata  #1182

### PR DESCRIPTION
**Pull request by agbeltran**

### CODE REVIEW

**Comment by riccardoAlbertoni** [comment]
This PR relates to #1182

**Comment by andrea-perego** [comment]
See also PR https://github.com/w3c/dxwg/pull/1209

**Comment by andrea-perego** [comment]
@riccardoAlbertoni , I'm fine with the revisions, but if I'm not mistaken (and I can be) the version on GitHub should always be in ED status, and the change to WD ... REC should be done offline or in a branch before publication.

Should this be the case, maybe we can just add the link to the errata document, and not change the specStatus.

**Comment by riccardoAlbertoni** [comment]
> @riccardoAlbertoni , I'm fine with the revisions, but if I'm not mistaken (and I can be) the version on GitHub should always be in ED status, and the change to WD ... REC should be done offline or in a branch before publication.
> 
> Should this be the case, maybe we can just add the link to the errata document, and not change the specStatus.

Thanks, Andrea, 
I do not remember, so in the doubt, I think we can take your suggestion as the safest approach;)

**Comment by agbeltran** [comment]
@riccardoAlbertoni not completely sure about the process, but if @andrea-perego is right and the spec should remain on ED, do you want me to change that and only leave the URL to the errata document? or we close this PR and we add the URL in the other PR about the errata.

File: dcat/config.js
Comment (riccardoAlbertoni) [inline dcat/config.js]: As suggested by Andrea, we keep the ED status 
```suggestion
    specStatus: "ED",
```

**Review by riccardoAlbertoni — APPROVED** [review]

**Comment by riccardoAlbertoni** [comment]
> @riccardoAlbertoni not completely sure about the process, but if @andrea-perego is right and the spec should remain on ED, do you want me to change that and only leave the URL to the errata document? or we close this PR and we add the URL in the other PR about the errata.

I have already implemented the change in this PR as suggested by Andrea. So I think we can simply approve this and merge it. 
My apologies for changing in your PR, I realize now that is probably rude from me ..

**Comment by agbeltran** [comment]
that's great @riccardoAlbertoni :-) I now updated w.r.t. the ```gh-pages``` branch, so waiting for another review

**Review by dr-shorthair — APPROVED** [review]

**Pull request merged**
